test/decl/protocol/special/coding/Inputs/class_codable_simple_multi1.swift

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: objc_interop
```

Comment:
Doug mentioned this too. Keep in mind though that these protocols use `Data`, which may pull in other things which really do require ObjC; this may prohibit us from being able to mock this in a truly `objc_interop`-free way...